pysindy/pysindy.py

```diff
@@ -91,18 +91,21 @@ class SINDy(BaseEstimator):
     0.999999985520653
     """
 
-    def __init__(
-        self,
-        optimizer=STLSQ(),
-        feature_library=PolynomialFeatures(),
-        differentiation_method=FiniteDifference(),
-        feature_names=None,
-        discrete_time=False,
-        n_jobs=1,
-    ):
-        self.optimizer = optimizer
-        self.feature_library = feature_library
-        self.differentiation_method = differentiation_method
+    def __init__(self, optimizer=None, feature_library=None,
+                 differentiation_method=None, feature_names=None,
+                 discrete_time=False, n_jobs=1):
```

Comment:
The (black)[https://black.readthedocs.io/en/stable/] formatter that's triggered when we push code will want this reformatted as follows:
```suggestion
    def __init__(
        self,
        optimizer=None,
        feature_library=None,
        differentiation_method=None,
        feature_names=None,
        discrete_time=False,
        n_jobs=1,
    ):
```